moveit_simple/src/moveit_simple.cpp

```diff
@@ -74,63 +74,50 @@ Robot::Robot(const ros::NodeHandle & nh, const std::string &robot_description,
 
 
 
-bool Robot::addTrajPoint(const std::string & traj_name, const Eigen::Affine3d pose,
+void Robot::addTrajPoint(const std::string & traj_name, const Eigen::Affine3d pose,
                   const std::string & frame, double time,
                   const std::string & point_name)
 {
   std::lock_guard<std::recursive_mutex> guard(m_);
 
-  bool success = false;
   ROS_INFO_STREAM("Attempting to add " << point_name << " to " << traj_name << "relative to"
                   << frame << " at time " << time);
-  try
-  {
     Eigen::Affine3d pose_rel_robot = transformToBase(pose, frame);
     std::unique_ptr<TrajectoryPoint> point =
       std::unique_ptr<TrajectoryPoint>(new CartTrajectoryPoint(pose_rel_robot, time, point_name));
-    success = addTrajPoint(traj_name, point);
-  }
-  catch (tf2::TransformException &ex)
-  {
-    ROS_WARN_STREAM("Add to trajectory failed for arbitrary pose point: " << ex.what());
-    success = false;
-  }
-  return success;
-
+    addTrajPoint(traj_name, point);
 }
 
 
 
 
-bool Robot::addTrajPoint(const std::string & traj_name, const std::string & point_name,
+void Robot::addTrajPoint(const std::string & traj_name, const std::string & point_name,
                          double time)
 {
   std::lock_guard<std::recursive_mutex> guard(m_);
 
   ROS_INFO_STREAM("Attempting to add " << point_name << " to " << traj_name
                  << " at time " << time);
   std::unique_ptr<TrajectoryPoint> point = lookupTrajectoryPoint(point_name, time);
```

Comment:
You should throw an exception from here.  This is the point at which you know why things failed.  The `point_name` is not valid.  The null pointer exception that would get thrown in the next function call isn't useful if you don't know why a null pointer was passed.  In fact it's confusing, because the user never passed a pointer to the method.